README.md

```diff
@@ -1,8 +1,4 @@
 # Software
-<!--
```

Comment:
nice! i've been meaning to remove `.appveyor.yaml` but I keep forgetting, can you remove that too as part of this PR?

Reply:
Good call! Removed.